rubin_sim/maf/batches/scienceRadarBatch.py

```diff
@@ -296,6 +296,17 @@ def scienceRadarBatch(colmap=None, runName='opsim', extraSql=None, extraMetadata
                                  displayDict=displayDict, mapsList=[stellar_map])
         bundleList.append(bundle)
 
+    # Brown Dwarf Volume
+    displayDict['subgroup'] = 'Brown Dwarf'
+    slicer = slicers.HealpixSlicer(nside=nside)
+    sum_stats = [metrics.VolumeSumMetric(nside=nside)]
```

Comment:
Are there other summary stats that would be useful? The VolumeSum is clearly useful to show the total volume that is reachable and this should correlate with the total number of brown dwarfs I think .. but maybe also the mean/median/max distance might be useful?

Reply:
Are they ok with only running this for the equivalent of the L7 brown dwarfs?